### ISSUE TITLE
class not found on import statement

### ISSUE THREAD

**Issue by hitdrumhard** [bharatraj88/angular2-timepicker#1]
import {TimePickerComponent} from "angular2-timepicker/timepicker-component";

statement show path not found.

**Comment by ghost** [comment]
Hey guys, Do you have any update on this?
Also I've tried to import the module without success too:

import { TimepickerModule } 		from "angular2-timepicker/src/timepicker";
import { TimepickerModule } 		from "angular2-timepicker/src";
import { TimepickerModule } 		from "angular2-timepicker";

Module not found: Error: Can't resolve 'angular2-timepicker/src/timepciker'
Module not found: Error: Can't resolve 'angular2-timepicker/src'
Module not found: Error: Can't resolve 'angular2-timepicker'

Can anyone let me know how to solve that?

Thanks

**Comment by ghost** [comment]
I've found the problem, there's a missing file index.ts in src/ path 
 you'll need to add a new index.ts file inside of src and add the following code:

export { TimepickerModule } from './timepicker.module';

Can you do that? I'll appreciate that change.

**Comment by nitishkumarnitc** [comment]
How can I install it

**Comment by KissBalazs** [comment]
would be nice to implement what @bruno-serfe  said, this issue still stands.

**Comment by bharatraj88** [comment]
@KissBalazs

Yes I have missed that. I will correct it in the next release. Meanwhile try to import from the src folder

**Comment by andela-mallan** [comment]
It would have been nicer if **@bruno-serfe** suggestion was implemented by now.
Or an alternative of how to import from the src folder, does the import statement change? 
**_Issue still stands_**. Thanks

**Comment by msms007** [comment]
@bharatraj88 
Could u pls clarify why jquery-ui is required as a dependency? while the jquery-timepicker works without that. I see the only dependency is jquery. 
A working plunker is here without jq-ui
https://jsfiddle.net/mogisum007/p4ge8e99/2/
